Makefile

```diff
@@ -55,9 +55,16 @@ test-setup-e2e: dev
 # test-e2e-cirecleci does e2e test setup and then runs the e2e tests
 test-e2e-cirecleci: test-setup-e2e
 	@echo "==> Testing ${NAME} (e2e)"
-	@go test ./e2e -race -count=1 -timeout=900s -tags=e2e ./... ${TESTARGS}
+	@echo "Tests regex: $(shell cat ".build/chunks/chunk.${CHUNK_INDEX}")"
+	@gotestsum --format testname --junitfile .build/test-results/e2e-tests-${CHUNK_INDEX}.xml -- \
+		./e2e -race -count=1 -timeout=600s -tags=e2e -run="$(shell cat ".build/chunks/chunk.${CHUNK_INDEX}")" ${TESTARGS}
 .PHONY: test-e2e-cirecleci
 
+test-unit-cirecleci:
```

Comment:
TODO: Get rid of this and make the gotestsum in integration test make target

Reply:
Addressed in https://github.com/hashicorp/consul-terraform-sync/pull/869/commits/b6e8e0ee61a8137207278705c7c473ebb52d612e